docs/madr/decisions/assets/038/all-dd.txt

```diff
@@ -0,0 +1,328 @@
+envoy_access_logs_grpc_access_log_logs_dropped_count
```

Comment:
Can we put these in gist to avoid adding 100s of lines to this repo?

Reply:
Yes, I'll add all of them to a gist